shell/src/main/java/alluxio/cli/fs/command/LsCommand.java

```diff
@@ -192,47 +218,35 @@ private void ls(AlluxioURI path, boolean recursive, boolean forceLoadMetadata, b
     if (forceLoadMetadata) {
       options.setLoadMetadataType(LoadMetadataType.Always);
     }
-    List<URIStatus> statuses = listStatusSortedByIncreasingCreationTime(path, options);
-    for (URIStatus status : statuses) {
+    List<URIStatus> statuses = mFileSystem.listStatus(path, options);
+    List<URIStatus> sorted = sortBySortField(statuses, sortField);
+    for (URIStatus status : sorted) {
       if (!pinnedOnly || status.isPinned()) {
         printLsString(status, hSize);
       }
       if (recursive && status.isFolder()) {
         ls(new AlluxioURI(path.getScheme(), path.getAuthority(), status.getPath()), true,
-            forceLoadMetadata, false, hSize, pinnedOnly);
+            forceLoadMetadata, false, hSize, pinnedOnly, sortField);
       }
     }
   }
 
-  private List<URIStatus> listStatusSortedByIncreasingCreationTime(AlluxioURI path,
-      ListStatusOptions options) throws AlluxioException, IOException {
-    List<URIStatus> statuses = mFileSystem.listStatus(path, options);
-    Collections.sort(statuses, new Comparator<URIStatus>() {
-      @Override
-      public int compare(URIStatus status1, URIStatus status2) {
-        long t1 = status1.getCreationTimeMs();
-        long t2 = status2.getCreationTimeMs();
-        if (t1 < t2) {
-          return -1;
-        }
-        if (t1 == t2) {
-          return 0;
-        }
-        return 1;
-      }
-    });
-    return statuses;
+  private List<URIStatus> sortBySortField(List<URIStatus> statuses, String sortField) {
+    return statuses.stream()
+            .sorted(SORT_FIELD_COMPARATORS.getOrDefault(sortField,
+                    SORT_FIELD_COMPARATORS.get("creationTime")))
+            .collect(Collectors.toList());
   }
 
   @Override
   public void runCommand(AlluxioURI path, CommandLine cl) throws AlluxioException, IOException {
     ls(path, cl.hasOption("R"), cl.hasOption("f"), cl.hasOption("d"), cl.hasOption("h"),
-        cl.hasOption("p"));
+        cl.hasOption("p"), cl.getOptionValue("sort", "creationTime"));
   }
 
   @Override
   public String getUsage() {
-    return "ls [-d|-f|-p|-R|-h] <path>";
+    return "ls [-d|-f|-p|-R|-h|--sort=WORD] <path>";
```

Comment:
WORD -> option?

Reply:
I can rename it , just went after what ubuntu/linux ls help gave me :)

Reply:
oh, I see. I actually checked `ls help` on mac, but it seems different :)